.github/workflows/compilation_on_android_ubuntu.yml

```diff
@@ -392,6 +392,10 @@ jobs:
             $THREADS_TEST_OPTIONS,
             $WASI_TEST_OPTIONS,
           ]
+        wasi_sdk_release:
+          [
+            "https://github.com/WebAssembly/wasi-sdk/releases/download/wasi-sdk-12/wasi-sdk-12.0-linux.tar.gz",
+          ]
```

Comment:
We have upgraded this dependency to wasi-sdk-19 since 27e7e160af54fe8a74620c4fbfacad480b5cb00c. Please rebase.

Reply:
Sure, thanks!